tests/cxx/no_fwd_decls.cc

```diff
@@ -29,6 +29,10 @@ IndirectClass* global;
 class LocalFwd;
 void ForwardDeclareUse(const LocalFwd*);
 
+// IWYU must not remove the forward declaration of this class even though its
+// definition can be found in the same file but after the use.
+class LocalFwd {};
+
```

Comment:
Does the existing test below with `Fwd` and `ForwardDeclareUse` cover the new "find-first-decl-not-in-this-file" logic, or does it need a separate test case? It seems to me this should cover it.

Reply:
You are right, it is not necessary.